Gemfile

```diff
@@ -0,0 +1,74 @@
+source 'https://rubygems.org'
+
+# Looking to use the Edge version? gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.0.1'
+
+# Use Puma as the app server
+gem 'puma', '~> 3.0'
+
+# Use Rack Timeout. Read more: https://github.com/heroku/rack-timeout
+gem 'rack-timeout', '~> 0.4'
+
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 2.5'
+
+# Use PostgreSQL as the database for Active Record
+gem 'pg', '~> 0.18'
+
+# Use Redis Rails to set up a Redis backed Cache and / or Session
+gem 'redis-rails', '~> 5.0.0.pre'
+
+# Use Sidekiq as a background job processor through Active Job
+gem 'sidekiq', '~> 4.2'
+
+# Use Clockwork for recurring background tasks without needing cron
+# gem 'clockwork', '~> 2.0'
+
+# Use Kaminari for pagination
+# gem 'kaminari', '~> 0.16'
+
+# Use SCSS for stylesheets
+gem 'sass-rails', '~> 5.0'
+
+# Use Uglifier as the compressor for JavaScript assets
+gem 'uglifier', '>= 1.3.0'
+
+# Use jQuery as the JavaScript library
+gem 'jquery-rails'
+
+# Use Turbolinks. Read more: https://github.com/turbolinks/turbolinks
+gem 'turbolinks', '~> 5'
+
+# Use Bootstrap SASS for Bootstrap support
+gem 'bootstrap-sass', '~> 3.3'
```

Comment:
Hmm... it might be an easy way to make our config page look nice. If we want to dump it I'm fine though.